website/docs/r/private_endpoint.html.markdown

```diff
@@ -199,7 +199,9 @@ An `ip_configuration` supports the following:
 
 * `private_ip_address` - (Required) Specifies the static IP address within the private endpoint's subnet to be used. Changing this forces a new resource to be created.
 
-* `subresource_name` - (Required) Specifies the subresource this IP address applies to. `subresource_names` corresponds to `group_id` and in this context is also used for `member_name`. Changing this forces a new resource to be created.
+* `subresource_name` - (Required) Specifies the subresource this IP address applies to. `subresource_names` corresponds to `group_id`. Changing this forces a new resource to be created.
```

Comment:
and this also is group id?

Reply:
by swagger, the `subresource_name`  should be alias to group_id [specs](https://github.com/Azure/azure-rest-api-specs/blob/b41f929626289b59e31be8a1091c99994864b096/specification/network/resource-manager/Microsoft.Network/stable/2021-05-01/privateEndpoint.json#L1019), and member_name is the name of the member of group_id which should not reuse the `subresource_name` value.

Reply:
i'm sorry but i don't follow, could you try and explain that again? i don't see subresource_name there at all even

Reply:
there are three properties `privateIPAddress`, `memberName` and `groupId` defined in swagger for `PrivateEndpointIPConfigurationProperties`, but only two properties defined in `azurerm` in the first place, they are `private_ip_address` (for `privateIPAddress`) and `subresource_name` (for both `groupId` and `memberName`). I believe the `subresource_name` defined in the azurerm schema is a rename of `groupId`. this PR is adding a new property for `memberName` to distinguish `groupId` and `memberName` in the request.